app/views/check_your_answers/_non_degree_qualifications.erb

```diff
@@ -0,0 +1,32 @@
+<h2 class='govuk-heading-m'><%= t('application_form.qualification_section.heading') %></h2>
+  <div id='qualification-list'>
+  <table class='govuk-table govuk-!-font-size-16'>
+    <thead class='govuk-table__head'>
+    <tr class='govuk-table__row'>
+      <th class='govuk-table__header' scope='col' width='30%'>Degree(s)</th>
+      <th class='govuk-table__header' scope='col' width='10%'>Grade</th>
+      <th class='govuk-table__header' scope='col'>Institution</th>
+      <th class='govuk-table__header' scope='col'>Year awarded</th>
+      <th class='govuk-table__header govuk-!-padding-right-2' scope='col'>
+        <span class='govuk-visually-hidden'>Actions</span>
+      </th>
+    </tr>
+    </thead>
+    <tbody class='govuk-table__body'>
+
+    <% qualifications.each do |qualification| %>
```

Comment:
This table should include

- Qualification: {type} {subject} eg GCSE Maths
- Grade
- Institution
- Year awarded